### ISSUE TITLE
Case timeline recommendations

### ISSUE THREAD

**Issue by laurjpeterson** [department-of-veterans-affairs/caseflow#9382]
TBD

**Comment by eric-nava** [comment]
According to Lit Support special counsel, there is a need for a printable version of the case timeline.

**Comment by laurjpeterson** [comment]
@eric-nava - did they say why? what problem does it solve?

**Comment by sneha-pai** [comment]
they need a way to print this out physically because when they physically submit this data via a packet to CAVC, it becomes a proof of dates on the case. (currently they print out a VACOLS screen of prior locations)

**Comment by sneha-pai** [comment]
food for thought (via @aroltsch in [slack thread](https://dsva.slack.com/archives/C6E41RE92/p1551122699181400): 
**Anya Roltsch**
Have we given any thought to expand case timeline for legacy appeals?
**Anya Roltsch**
One of the Alex's suggestions per his DAS deprecation tech spec
**Anya Roltsch**
![image 37](https://user-images.githubusercontent.com/31224257/53363441-76f90c00-390a-11e9-8dc8-1ea23d659595.png)
**Lauren Peterson**
 not recently, but yes, have thought about it before. i wondered whether we wanted to use any appeals status stuff for legacy instead of using VACOLS locations but didn't prioritize it because of AMA work

**Comment by allyceh** [comment]
Initial proposal is in Mural and out for feedback. In Mural, search: "Case Timeline Refresh Designs".

**Comment by allyceh** [comment]
Note for the team: While onsite at the Columbia RO today, the Booz Allen team showed me POA history field in VBMS. Everytime POA is changed, VBMS displays the date, time, and user who changed it. 

I am not sure if this is a VBMS attribute that we'd want to pull in Caseflow? Or, just track POA tasks that happen at BVA? The data is there now if we want to investigate.

**Comment by laurjpeterson** [comment]
Yes yes we definitely should! Could you find the API details or a contact person that an engineer on our side can work with? This might be new data that VBMS is now surfacing.

**Comment by allyceh** [comment]
Assigning to @sneha-pai to update ticket based on usability testing and define user stories for engineering.

**Comment by laurjpeterson** [comment]
Adding most recent [Mural](https://app.mural.co/t/workqueue2001/m/workqueue2001/1556035219909/9317bc82fd2cf504e3e8539a172a33750c406a0d) with new designs. I think this Mural covers all items in the original ticket. We might be able to close this ticket in favor of other development tickets (to be created after we discuss with engineering). @sneha-pai what do you think?

**Comment by mbaenatan** [comment]
Per prioritization discussion with engineering and product, we should consolidate all case timeline tickets and create an Epic for case timeline improvements.
